src/http/server/handlers/mod.rs

```diff
@@ -47,11 +47,21 @@ impl<B: Send> FromRequest<B> for QueryConfigFromReq {
             #[cfg(any(test, feature = "cli", feature = "test-fixture"))]
             QueryType::TEST_MULTIPLY_STR => Ok(QueryType::TestMultiply),
             QueryType::IPA_STR => {
-                // TODO: these are hard-coded, but should be retrieved from the request
+                #[derive(serde::Deserialize)]
+                struct IPAQueryConfigParam {
+                    num_bits: u32,
+                    per_user_credit_cap: u32,
+                    max_breakdown_key: u128,
```

Comment:
we probably need to task Taiki about actual arguments for IPA query. I feel like `max_breakdown_key` shouldn't be the input for a query, but could be wrong here